### ISSUE TITLE
Provide instructions on mouse over in exhibit

### ISSUE THREAD

**Issue by ghost** [scholarslab/Neatline#171]
When I got to an exhibit page, I didn't know how to scroll the timeline. Something like "click to scroll" might help.

**Comment by waynegraham** [comment]
Are you talking a piece of text on the interface, or when a user hovers over the timeline, there's some type of dialog that appears?

**Comment by davidmcclure** [comment]
Yep, I think this is a good idea. I think this would be especially useful in the editor - I imagine a collection of small, subtle "?" icons next to editing controls that you could click on and get a 2-3 sentence description of what the control does. This could also be implemented in the public-facing interface.

**Comment by ghost** [comment]
I hadn't even thought about the editor (but that sounds great, too). I was thinking about the exhibits themselves. When I went poking around one of the exhibits with Jeremy, he told me to scroll through the timeline, and it took me about 10 seconds to realize what he meant--click to drag the timeline. If I had been alone I might not have even realized the timeline was interactive. Simple instructions that pop up when you hover over each element would make this way more intuitive for the user.
